klippy/kinematics/cartesian.py

```diff
@@ -124,6 +124,11 @@ def move(self, print_time, move):
         for i, rail in enumerate(self.rails):
             if move.axes_d[i]:
                 rail.step_itersolve(move.cmove)
+    def get_status(self):
+        return { 'homed': tuple(
+                    self.limits[i][0] <= self.limits[i][1] for i in (0, 1, 2)),
```

Comment:
I think it would be nice to eventually export this info to gcode templates.  So, I think exporting as a string would be easier to document.  On cartesian and corexy this could be something like:
```
return { 'homed_axes': "".join([a for a, (l, h) in zip("XYZ", self.limits) if l <= h]) }

Reply:
I removed the limits part and changed the format of homed_axes as you suggested!